packages/gp2-components/src/organisms/OutputRelatedResearchCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { ResearchOutputRelatedResearchCard } from '@asap-hub/react-components';
+import { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { ComponentProps } from 'react';
+import {
+  outputArticle,
+  outputCode,
+  outputDataset,
+  outputForm,
+  outputMaterial,
+  outputReport,
```

Comment:
The icons are bigger than the space they should fit into

<img width="1404" alt="Screenshot 2023-10-06 at 07 52 46" src="https://github.com/yldio/asap-hub/assets/16595804/1df18c7e-c0f8-46c2-887c-5e916ca19856">

I think adding viewBox to the svgs will fix it :)